doc/rbd/rbd-nomad.rst

```diff
@@ -52,20 +52,24 @@ diagram depicts the Nomad/Ceph technology stack.
 Create a Pool
 =============
 
-By default, Ceph block devices use the ``rbd`` pool. Create a pool for
-Nopmad persistent storage. Ensure your Ceph cluster is running, then create
-the pool. ::
+By default, Ceph block devices use the ``rbd`` pool. Create a pool for Nomad
+persistent storage. Ensure that your Ceph cluster is running, then create the
+pool:
 
-        $ ceph osd pool create nomad
+.. prompt:: bash #
+
+  ceph osd pool create nomad
 
 See `Create a Pool`_ for details on specifying the number of placement groups
-for your pools, and `Placement Groups`_ for details on the number of placement
+for your pools. See `Placement Groups`_ for details on the number of placement
 groups you should set for your pools.
 
 A newly created pool must be initialized prior to use. Use the ``rbd`` tool
-to initialize the pool::
+to initialize the pool:
+
+.. prompt:: bash #
```

Comment:
ditto.

Reply:
This suggestion has been incorporated into the text.